Ladybird/Qt/WebContentView.h

```diff
@@ -42,7 +42,7 @@ class WebContentView final
     , public WebView::ViewImplementation {
     Q_OBJECT
 public:
-    WebContentView(WebContentOptions const&, StringView webdriver_content_ipc_path, RefPtr<WebView::WebContentClient> parent_client = nullptr, size_t page_index = 0);
+    WebContentView(QWidget* window, WebContentOptions const&, StringView webdriver_content_ipc_path, RefPtr<WebView::WebContentClient> parent_client = nullptr, size_t page_index = 0);
```

Comment:
This will engage the Qt lifetime properties, right? iirc the WebContentView is kept alive in another way

Reply:
I compiled Ladybird with ASan and it doesn't indicate any issues other than that whole document leak that is also sometimes happens on CI even without this PR. I think Qt is just smart enough to understand what's going on :)

Also, QLayout doesn't inherit from QWidget, so it probably can't be an owner of anything? (I might be totally wrong here, have no idea how Qt works). That said, I switched parent of the main WebContentView to be Tab and not BrowserWindow.

Reply:
:shrug: all right. if it works it works.